docs/Consuming.md

```diff
@@ -82,11 +91,13 @@ await consumer.run({
 })
 ```
 
-* `batch.highWatermark` is the last committed offset within the topic partition. It can be useful for calculating lag.
 * `eachBatchAutoResolve` configures auto-resolve of batch processing. If set to true, KafkaJS will automatically commit the last offset of the batch if `eachBatch` doesn't throw an error. Default: true.
+* `batch.highWatermark` is the last committed offset within the topic partition. It can be useful for calculating lag.
 * `resolveOffset()` is used to mark a message in the batch as processed. In case of errors, the consumer will automatically commit the resolved offsets.
-* `commitOffsetsIfNecessary(offsets?)` is used to commit offsets based on the autoCommit configurations (`autoCommitInterval` and `autoCommitThreshold`). Note that auto commit won't happen in `eachBatch` if `commitOffsetsIfNecessary` is not invoked. Take a look at [autoCommit](#auto-commit) for more information.
+* `async heartbeat()` can be used to send heartbeat to the broker according to the set `heartbeatInterval` value in consumer [configuration](#options).
```

Comment:
Maybe instead of `async` we should replace this with a Typescript-esque type definition, since that's a standard way of expressing this ex. `heartbeat(): Promise<void>`

Reply:
done 👍